ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -8015,6 +8015,42 @@ def test_torch_instance_sign(
     )
 
 
+# std
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="std",
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
This strategy will generate unstable examples you can use this one as it is already implemented to handle statistical values https://github.com/unifyai/ivy/blob/0a32c763fa1d7fd2fd5c9803b31eeb23752e2947/ivy_tests/test_ivy/test_functional/test_core/test_statistical.py#L12